crnk-core/src/main/java/io/crnk/core/engine/internal/utils/PropertyUtils.java

```diff
@@ -76,6 +78,10 @@ public static Type getPropertyType(Class<?> beanClass, String field) {
 		return INSTANCE.findPropertyType(beanClass, field);
 	}
 
+	public static Class<?> getRawPropertyType(Class<?> beanClass, String field) {
```

Comment:
could you rely here on BeanInformation.of(...) rather than introducing more methods to this class? Then we could maybe sooner rather than later get rid of it... :-)

Reply:
I do not understand what you mean by `BeanInformation.of(...)`. I cannot find such a method.

Reply:
ah, sorry BeanInformation.get(...)